be/src/pipeline/exec/exchange_sink_buffer.cpp

```diff
@@ -470,18 +424,13 @@ bool ExchangeSinkBuffer::_is_receiver_eof(InstanceLoId id) {
     return _instance_to_receiver_eof[id];
 }
 
-void ExchangeSinkBuffer::_turn_off_channel(InstanceLoId id, bool cleanup) {
+void ExchangeSinkBuffer::_turn_off_channel(InstanceLoId id) {
```

Comment:
if we call the function, we must get the lock of `std::unique_lock<std::mutex> lock(*_instance_to_package_queue_mutex[id]);` so better we can read the important msg from comment or function name